src/routes/LandingPage/LandingPage.tsx

```diff
@@ -173,6 +181,26 @@ export default function LandingPage(): ReactElement {
         </FeatureHighlightsContainer>
       </ContentContainer>
 
+      <TutorialVideoContainer>
+        <ContentContainer style={{ gap: 20 }}>
+          <h2 style={{ margin: 0 }}>See how Timelapse Feature Explorer can help in your research and analysis</h2>
+          <VideoList videoEntries={LANDING_PAGE_VIDEO_CONTENT}></VideoList>
+          <h2 style={{ margin: 0 }}>Additional resources</h2>
+          <ul style={{ margin: 0 }}>
+            <li style={{ margin: 0 }}>
+              Follow{" "}
+              <ExternalLink
+                href={
+                  "https://github.com/allen-cell-animated/colorizer-data/blob/main/documentation/getting_started_guide/GETTING_STARTED.ipynb"
+                }
+              >
+                our data conversion tutorial on GitHub
+              </ExternalLink>
+            </li>
+          </ul>
```

Comment:
This is also subject to change pending Ed team input

<img width="403" height="121" alt="Image" src="https://github.com/user-attachments/assets/b83d153c-10ec-4374-8740-016b49018d02" />